packages/output/src/browser/output-contribution.ts

```diff
@@ -136,13 +145,30 @@ export class OutputContribution extends AbstractViewContribution<OutputWidget> i
             isVisible: widget => this.withWidget(widget, output => output.isLocked),
             execute: () => this.widget.then(widget => widget.unlock())
         });
+        registry.registerCommand(OutputCommands.COPY_ALL, {
+            execute: () => {
+                this.widget.then(async widget => {
+                    this.clipboardService.writeText(widget.getText());
+                });
+            }
+        });
+    }
+
+    registerKeybindings(registry: KeybindingRegistry): void {
+        registry.registerKeybinding({
+            command: OutputCommands.COPY_ALL.id,
+            keybinding: 'ctrlcmd+shift+c'
```

Comment:
@salkin-naj we should only register the keybinding when the view has focus else it will override all other commands with <kbd>ctrlc</kbd>+<kbd>shift</kbd>+<kbd>c</kbd> which is not the behavior we would want or expect.

We can do so by implementing a `when` context similarly to how it is done in other views:
- https://github.com/eclipse-theia/theia/blob/master/packages/scm/src/browser/scm-context-key-service.ts#L20-L46

Please let me know if you require help, I could assist you :)

Reply:
Everybody can do <kbd>Ctrl/⌘</kbd>+<kbd>A</kbd> then <kbd>Ctrl/⌘</kbd>+<kbd>C</kbd>. We do not need a keybinding for this. Please remove.